src/main/java/seedu/address/model/Model.java

```diff
@@ -50,6 +50,22 @@ void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
     /** Removes the specific tag (from all persons with that tag) */
     void removeTag(Tag tags) throws DuplicatePersonException, PersonNotFoundException;
 
```

Comment:
Some methods in `Model` interface are defined twice. Please fix.

Reply:
`updatePerson` method is still defined twice in `Model` interface

Reply:
See Travis CI error message at [here](https://travis-ci.org/CS2103AUG2017-T09-B3/main/builds/291463094?utm_source=github_status&utm_medium=notification), `checkStyle` does not pass due to trailing whitespace.